### ISSUE TITLE
Release K2 compatible version

### ISSUE THREAD

**Issue by 3flex** [drewhamilton/Poko#259]
K2 support was added in https://github.com/drewhamilton/Poko/pull/223

Is there any chance a new release could be issued please now the Kotlin 2.0 release cycle has begun with beta 1?

Currently I get this in a Kotlin/JVM project using 2.0.0-Beta1:

```
e: There are some plugins incompatible with language version 2.0:
  dev.drewhamilton.poko.PokoCompilerPluginRegistrar
Please use language version 1.9 or below
```

**Comment by drewhamilton** [comment]
Can do! I'm afk this week for the holidays so it'll be next week. Does the current snapshot work as expected?

**Comment by 3flex** [comment]
0.16.0-SNAPSHOT works as expected.

**Comment by drewhamilton** [comment]
Just released [0.15.1](https://github.com/drewhamilton/Poko/releases/tag/0.15.1).
